common/lib/api-client/middleware/cache-key.js

```diff
@@ -0,0 +1,30 @@
+const { get } = require('lodash')
+
+const models = require('../models')
+
+function cacheKeyMiddleware({ apiVersion = 2 } = {}) {
```

Comment:
Not sure it's a good idea to set the default API version here as well as in the configuration.

I think it would be better to not include a default here, and if we want one, it should be defined in the configuration file.

Reply:
👍

Reply:
Should it throw if not passed a version?

Reply:
Would that end up being an error on startup? That would be more clear than a silent failure and continuing.

Reply:
Good point

Reply:
>Should it throw if not passed a version?

I doubt this method should care. It's just to build a key, right? If not version, it just doesn't include that within the key.

And as @merlinc suggests, if our app **needs** a version, we should throw an error on startup, or set a default in the config.